src/server/index.js

```diff
@@ -85,6 +87,11 @@ function startServer () {
   async function handleRequest (req, res) {
     try {
       // parse address
+      // can parse for network here.
```

Comment:
server must have an explicit allowlist (not a denylist) for network ids it permits to send from. that faucet occasionally accidentally holds real world assets on mainnet and bsc

Reply:
@kumavis Got it, what does that mean for the parsing here?